docs/releasenotes/archive/service/year2017.md

```diff
@@ -6,3 +6,460 @@ description: This is an archive of 2017 Sumo Logic service release notes.
+Update - Metrics Introduction now references new UI layout, error messages updated, and revised and improved metrics query performance.
+Bug Fix - Aggregate group-by key is no longer case sensitive.
+Bug Fix -  You now go to the metrics page after clicking "Exit Setup Wizard" if you are working with a metrics source.
+Bug Fix - Division by zero not shown in metrics time series to improve visualizations.
+Bug Fix - Dashboards shared with“World” now open in incognito mode.    
+Bug Fix - Removing filter refreshes screen. 
+
+---
+## November 21, 2017 (Search) 
+
+Update - Improved error messages for exporting a search.
+
+---
+## November 21, 2017 (User Interface) 
+
+Update - Country and states list updated for Activate Your Account.
+Bug Fix - SAML sign-in redirects user to correct page on sign-out.
+Bug Fix - Show Password Policy only displays when user has Manage password policy capability.
+Bug Fix - Appropriate content shows when deleting a user.
+Bug Fix - Accounts page is now more exact.
+Bug Fix - Multiple tabs can now open during a search.
+Bug Fix - Error now shows when you attempt to name a dashboard, search, or folder with an existing name.
+Bug Fix - User role creation validates for invalid characters.
+
+---
+## November 7, 2017 
+
+There is a new app for Amazon Kinesis - Streams, updates to dashboards, new metrics operators, fix to sources, and some user interface changes as part of this update.
+
+---
+## November 7, 2017 (Apps)
+
+New Beta App - The Sumo Logic App for Amazon Kinesis - Streams is now available for Beta testing. This is a unified logs and metrics (ULM) App which provides information on the events and metrics of Kinesis Streams. The preconfigured dashboards help you monitor the events, API calls, errors, incoming and outgoing records, latencies, and throughput of Kinesis Streams.
+
+---
+## November 7, 2017 (Dashboards)
+
+Bug Fix - Dashboard with single panels no longer have display error when adding a new panel.
+Bug Fix - Dashboard changes now reflected on sidebar without a hard refresh.
+
+---
+## November 7, 2017 (Metrics)
+
+New Operators - New topk, bottomk, and filter operators available to filter your metrics query. You can now reduce your time series down and simplify your visuals with these operators. For example, you can find the top 10 time series with the highest average. 
+Topk Metrics  
+
+Bug Fix -  Drill down metrics name is now consistent with monitor name on dashboard panel.
+
+---
+## November 7, 2017 (Sources)
+
+Bug Fix - Docker Stats Source error message now displays correctly.
+
+---
+## November 7, 2017 (User Interface)
+
+Reminder -  Migrate to the new UI as soon as possible. The classic UI will no longer be available after Dec 15.
+Enhancement - Asterisk is now shown in the saved search tab to indicate that there are unsaved edits.    
+Enhancement - Scrollbar is now wider to make it easier to use.  
+Bug Fix - In-product notification icons now display correctly. 
```

Comment:
Needs to be a bulleted list

Reply:
It's done as a bulleted list in DocHub, but it's not needed as they are the release information that day. They are not sub-points.

Reply:
Ok! So here you'd just need to put line breaks between each. Otherwise they'll be lumped together as a giant paragraph.